src/shims/unix/freebsd/foreign_items.rs

```diff
@@ -63,6 +64,25 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 this.write_scalar(Scalar::from_target_usize(len, this), dest)?;
             }
 
+            // File related shims
+            "stat" | "stat@FBSD_1.0" => {
```

Comment:
So both of these exist? I would have expected only the `stat@FBSD_1.0` symbol.

Reply:
You have not responded to or resolved this comment.

Reply:
I m covering both cases like macos does.

Reply:
In macOS we did this because those symbols actually existed, but are deprecated.

So copying this only makes sense if the symbols also exist(ed) on FreeBSD.

Reply:
they do exist but from freebsd 12 and onwards.

Reply:
Okay, sounds good then. Please explain that in a comment.